pkg/cmd/shared/handler/indices/config.go

```diff
@@ -129,14 +129,18 @@ func ValidateImportConfigFlags(opts *ImportOptions) error {
 		return fmt.Errorf("%s Cannot clear existing synonyms if synonyms are not in scope", cs.FailureIcon())
 	}
 	// Scope and config
+	scopeError := make([]string, 0, 3)
 	if utils.Contains(opts.Scope, "settings") && opts.ImportConfig.Settings == nil {
-		return fmt.Errorf("%s No settings found in config file", cs.FailureIcon())
+		scopeError = append(scopeError, "settings")
 	}
 	if utils.Contains(opts.Scope, "rules") && len(opts.ImportConfig.Rules) == 0 {
-		return fmt.Errorf("%s No rule found in config file", cs.FailureIcon())
+		scopeError = append(scopeError, "rules")
 	}
 	if utils.Contains(opts.Scope, "synonyms") && len(opts.ImportConfig.Synonyms) == 0 {
-		return fmt.Errorf("%s No synonym found in config file", cs.FailureIcon())
+		scopeError = append(scopeError, "synonyms")
+	}
+	if len(scopeError) > 0 {
+		return fmt.Errorf("%s No %s found in config file", cs.FailureIcon(), utils.SliceToReadableString(scopeError))
```

Comment:
I wonder if we should err on this case, and not just output a warning that the missing scope would be ignored during the import (similar do what I did for the `dictionary entries clear` command). Thoughts?

Reply:
Yes because we have an impact summary before confirming the command so that makes sense.
However, maybe we should throw an error if the config will do nothing? 

For example: 
- scope to `settings, synonyms` but none of this exists in the config file => Error
- scope to `settings, synonyms`, settings exists in the config file but not synonyms => impact summary + confirm (no error)

wdyt?

Reply:
Agreed 👍